modules/notAchili/ss44UI/config/tip_conf.lua

```diff
@@ -0,0 +1,15 @@
+local ignoreweapon =
+{
+	armaak = {1},
```

Comment:
There is not such weapons... or any reference to these names...

Reply:
removed in [e1161ae]